s3/container.go

```diff
@@ -77,12 +80,12 @@ func (c *container) Items(prefix, cursor string, count int) ([]stow.Item, string
 	// If not, provide the file name of the last item as the next marker. S3 lists
 	// its items (S3 Objects) in alphabetical order, so it will receive the item name
 	// and correctly return the next list of items in subsequent requests.
-	marker := ""
+	startAfter := ""
 	if *response.IsTruncated {
-		marker = containerItems[len(containerItems)-1].Name()
+		startAfter = containerItems[len(containerItems)-2].Name()
```

Comment:
Thinking we should preform a check here to make sure that len-2 is okay. We may get an out of bounds error I want to protect against.

Reply:
Then return an error in the event it cannot be accessed.